fossunited/fossunited/doctype/foss_event_cfp/foss_event_cfp.py

```diff
@@ -199,11 +199,10 @@ def check_if_already_submitted(self):
 @frappe.whitelist()
 def create_cfp_submission(fields):
     fields_dict = {
-        "doctype": "FOSS Event CFP Submission",
         "submitted_by": frappe.session.user,
     }
     fields_dict.update(frappe.parse_json(fields))
-    return create_submission(fields_dict)
+    return create_submission("FOSS Event CFP Submission", fields_dict)
```

Comment:
Why a separate API is required for this? Why no used the out of the box rest API?

Reply:
Refactoring